# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1315066691 25200
#      Sat Sep 03 09:18:11 2011 -0700
# Node ID b37d8e5aedf381b84760ab3c47d72b5bbdecea10
# Parent  c5c94b63931f0e23691719bb4d5e6ce28f53328d
fntests.m: Remove statement about demos from "No Tests" message

* fntests.m: Remove statement about demos from "No Tests" message

diff --git a/test/fntests.m b/test/fntests.m
--- a/test/fntests.m
+++ b/test/fntests.m
@@ -210,17 +210,17 @@ function n = num_elts_matching_pattern (
   n = sum (cellfun (@(x) !isempty (x), regexp (lst, pat, 'once')));
 endfunction
 
 function report_files_with_no_tests (with, without, typ)
   pat = cstrcat ('\', typ, "$");
   n_with = num_elts_matching_pattern (with, pat);
   n_without = num_elts_matching_pattern (without, pat);
   n_tot = n_with + n_without;
-  printf ("\n%d (of %d) %s files have no tests or demos.\n", n_without, n_tot, typ);
+  printf ("\n%d (of %d) %s files have no tests.\n", n_without, n_tot, typ);
 endfunction
 
 pso = page_screen_output ();
 warn_state = warning ("query", "quiet");
 warning ("on", "quiet");
 try
   page_screen_output (0);
   warning ("off", "Octave:deprecated-functions");
